Example/PrimerSDK/CreateClientToken.swift

```diff
@@ -12,4 +12,5 @@ import Foundation
 struct CreateClientTokenRequest: Codable {
     let customerId: String
     let customerCountryCode: String?
+    var staging: Bool?
```

Comment:
Is this really needed?

Reply:
I think we should keep this, in case we want to target staging in the future

Reply:
Ok, now I get it. This is just going to add `?staging=true` on the URL, right?